substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/SystemPropertiesSupport.java

```diff
@@ -65,7 +72,36 @@
                     "java.specification.version",
                     "java.vm.specification.name",
                     "java.vm.specification.vendor",
-                    "java.vm.specification.version"
+                    "java.vm.specification.version",
+                    "sun.arch.data.model"
+    };
+
+    /**
+     * System properties are unsupported by default.
+     */
+    private static final String[] INCOMPATIBLE_PROPERTIES = {
```

Comment:
After thinking about the name for a while, my suggestion is to call it `UNDEFINED_PROPERTIES` (with the appropriate changes of the option and classes, fields, ...). That name avoids all sorts of legal hassles.

Reply:
fixed